packages/es-components/src/components/containers/tooltip/Tooltip.js

```diff
@@ -160,10 +164,13 @@ Popup.defaultProps = {
   style: {}
 };
 
+const SrContentContainer = screenReaderOnly(styled.div``);
```

Comment:
is there any reason you aren't using `screenReaderOnly('div'); ?

Reply:
Just... didn't know that was a thing. I was expecting `screenReaderOnly(<div />)` to work initially, but it didn't, so this was the quickest shorthand I knew to get to a function that returns a div.

Anyway, I'll switch it out.